app/init.php

```diff
@@ -1024,6 +1029,24 @@ function (mixed $value) {
     ]);
 }, []);
 
+App::setResource('queue', function () {
+
+    $fallbackForRedis = AppwriteURL::unparse([
```

Comment:
Maybe DSN library?

Reply:
This has been removed, looks like this is needed in redis sync PR. IT got here by mistake.